api/src/main/java/org/pmiops/workbench/survey/NewUserSatisfactionSurveyServiceImpl.java

```diff
@@ -22,13 +22,18 @@ public boolean eligibleToTakeSurvey(DbUser user) {
       return false;
     }
 
-    final Instant createdTime = user.getCreationTime().toInstant();
+    final Instant now = clock.instant();
+    return now.isAfter(eligibilityWindowStart(user)) && now.isBefore(eligibilityWindowEnd(user));
+  }
 
-    final Instant windowStart = createdTime.plus(2 * 7, ChronoUnit.DAYS);
-    final Instant windowEnd = windowStart.plus(61, ChronoUnit.DAYS);
+  private Instant eligibilityWindowStart(DbUser user) {
+    final Instant createdTime = user.getCreationTime().toInstant();
 
-    final Instant now = clock.instant();
+    return createdTime.plus(2 * 7, ChronoUnit.DAYS);
+  }
 
-    return now.isAfter(windowStart) && now.isBefore(windowEnd);
+  @Override
+  public Instant eligibilityWindowEnd(DbUser user) {
+    return eligibilityWindowStart(user).plus(61, ChronoUnit.DAYS);
```

Comment:
would 61 days change anytime before launching? should it be a config?

Reply:
I don't expect it to change before launching since the window was [decided a month ago](https://precisionmedicineinitiative.atlassian.net/browse/RW-9074) and I haven't heard anything about it changing. I also don't expect it to change after launching, since that could affect the content of the results. Both are entirely possible, of course!

I don't think this should be a config since I don't expect it to change and we don't need a different value per environment. I don't have much experience with what should and shouldn't be configurable though, so if you feel strongly about it I'll change it.

Reply:
sounds good, let's keep it then